vectorlm/utils/save_utils.py

```diff
@@ -171,13 +143,13 @@ def save_consolidated_model(
             torch.save(state_dict, save_path)
 
 
-def save_optimizer(
+def save_model_optimizer(
```

Comment:
A bit of a nitpick but can we name this something that is more expressive "save_model_and_optimizer" maybe?